Source/Scene/Cesium3DTileFeature.js

```diff
@@ -42,9 +42,11 @@ define([
      * }, Cesium.ScreenSpaceEventType.MOUSE_MOVE);
      */
     function Cesium3DTileFeature(tileset, content, batchId) {
+        this._content = content;
         this._batchTable = content.batchTable;
```

Comment:
For the sake of reducing the size of a feature even further, we could remove `_batchTable` too since this can be accessed from `content`.